tests/test_datasets.py

```diff
@@ -0,0 +1,129 @@
+import json
+import os
+import threading
+import unittest
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from test.support import EnvironmentVarGuard
+from urllib.parse import urlparse
+
+from kaggle_secrets import (_KAGGLE_URL_BASE_ENV_VAR_NAME,
+                            _KAGGLE_USER_SECRETS_TOKEN_ENV_VAR_NAME,
+                            CredentialError, BackendError, ValidationError)
+from kaggle_web_client import KaggleWebClient
+from kaggle_datasets import KaggleDatasets, _KAGGLE_TPU_NAME_ENV_VAR_NAME
+
+_TEST_JWT = 'test-secrets-key'
+
+_TPU_GCS_BUCKET = 'gs://kds-tpu-ea1971a458ffd4cd51389e7574c022ecc0a82bb1b52ccef08c8a'
```

Comment:
Just to double check: this isn't any sort of actual sensitive bucket right?

Reply:
Nope. These aren't even actual bucket names. I took a test (public) bucket and modified the name slightly to create these.